pom.xml

```diff
@@ -545,12 +545,24 @@
                             <user>1000</user>
                         </container>
                         <extraDirectories>
-                            <paths>src/main/docker/jib</paths>
+                            <paths>
+                                <path>
+                                    <from>src/main/docker/jib</from>
+                                    <into>/</into>
+                                    <excludes>
```

Comment:
Since .gitkeep is a temporary file, I don't see reason to include over here in registry. Can we just remove?